docker/Dockerfile

```diff
@@ -77,55 +77,56 @@ ARG CUDA_VERSION
 # If this file is present after the first command, kaniko
 # won't be able to build the docker image.
 RUN if [ -n "$CUDA_VERSION" ]; then \
-        rm -f /usr/local/cuda-$(echo $CUDA_VERSION | cut -c -4)/cuda-$(echo $CUDA_VERSION | cut -c -4); \
```

Comment:
Are these dockerfile changes needed? Looks like unnecessary formatting changes, should we revert?

Reply:
Only thing that should be changing here is manually installing snappy as it will not properly install normally

Reply:
Looks good, thanks!